#!/usr/bin/python

PACKAGE='jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("template_id", int_t, 1, "Template ID to Change", 0, 0, 5)
gen.add("frame_id", str_t, 0, "Selected Template Name", "")

gen.add("reprojection_threshold", double_t, 2, "",  3.0, 0, 30.0)
gen.add("distanceratio_threshold", double_t, 4, "", 0.80, 0, 1.0)
gen.add("error_threshold", double_t, 8, "",  50.0, 0, 2000.0)

exit(gen.generate(PACKAGE, "point_pose_extractor", "point_pose_extractor"))
